docs/_plugins/insert_token.rb

```diff
@@ -1,6 +1,6 @@
 module InsertToken
     def insert_token(input)
-        input.sub("<script>", "<script>\nmapboxgl.accessToken = '<your access token here>';")
+        input.sub("<script>", "<script>\n  mapboxgl.accessToken = '<your access token here>';")
```

Comment:
Ah ... the docs page is different (screenshot below). But instead of changing what I changed, we should probably just unindent the `map` line there so it matches the examples, right?

![screen shot 2016-06-28 at 2 17 49 pm](https://cloud.githubusercontent.com/assets/628431/16432696/2bdb406c-3d3b-11e6-9a39-1dd5c75d1672.png)